### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by LiamPClancy** [emilk/egui#1692] [General]
I just wanted to give a big thank you for all the contributors into this project. 
decided to learn rust last week, and coming from a C# and JS background there is a big learning curve, lots of half done repos that no longer work, and difficulty finding good examples.
this project and the associated templates, examples, and documentation - have allowed me make significant progress since finding it. 

Thank you, you guys are great. 
hopefully after I get my head around the tech a little be more i'll be able to pay it back by contributing.

**Comment by emilk** [comment]
Thank you ❤️

**Comment by sbechet** [comment]
Hello, I wanted to say the same thing: Thank you very much. it's ridiculously easy to create graphical applications with `egui`. You are magicians.
